src/components/vm/vmDetailsPage.jsx

```diff
@@ -87,28 +88,15 @@ export const VmDetailsPage = ({
     if (cockpit.location.path[1] == "console") {
         return (
             <WithDialogs key="vm-details">
-                <Page id={"vm-" + vm.name + "-consoles-page"}
-                      className="consoles-page-expanded no-masthead-sidebar">
-                    <PageBreadcrumb hasBodyWrapper={false} stickyOnBreakpoint={{ default: "top" }}>
-                        <Breadcrumb className='machines-listing-breadcrumb'>
-                            <BreadcrumbItem to='#'>
-                                {_("Virtual machines")}
-                            </BreadcrumbItem>
-                            <BreadcrumbItem to={'#' + cockpit.format("vm?name=$0&connection=$1", encodeURIComponent(vm.name), vm.connectionName)}>
-                                {vm.name}
-                            </BreadcrumbItem>
-                            <BreadcrumbItem isActive>
-                                {_("Console")}
-                            </BreadcrumbItem>
-                        </Breadcrumb>
-                    </PageBreadcrumb>
-                    {vmActionsPageSection}
-                    <PageSection isFilled hasBodyWrapper={false}>
-                        <Consoles vm={vm} config={config}
-                            onAddErrorNotification={onAddErrorNotification}
-                            isExpanded />
-                    </PageSection>
-                </Page>
+                <div id={"vm-" + vm.name + "-consoles-page"}
```

Comment:
Similar to f-strings we have [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) we can use too. Just FYI
```suggestion
                <div id={`vm-${vm.name}-consoles-page`}
```